groups/ecs/sg.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_security_group" "alb_security_group" {
+  name        = "${var.environment}-${var.service}-lb"
+  description = "Restricts access for ${var.service}-${var.environment} lb ${var.service} nodes"
+  vpc_id      = data.aws_vpc.placement.id
+
+  ingress {
+    description     = "lb HTTPS ingress from admin and concourse CIDRs"
```

Comment:
We'll have to update the description here as I don't think the Concourse CIDRs are included in the administration list